src/app/billingentity/billingentity-members/billing-entity-members.component.ts

```diff
@@ -258,19 +255,15 @@ export class BillingEntityMembersComponent implements OnInit, OnDestroy {
     });
     forkJoin(upsert$).subscribe({
       next: () => {
-        this.messageService.add({
-          severity: 'success',
-          summary: $localize`Successfully saved`,
-        });
+        this.notificationService.showSuccessMessage(
+          $localize`Successfully saved Billing ${payload.billingEntity.metadata.name}.`
```

Comment:
Maybe use the display name instead?
Note: Use the metadata name as fallback in case the display name is undefined or empty.

Reply:
As part of a different [ticket](https://github.com/appuio/cloud-portal/issues/492), I've added a pipe that uses either the `displayName` of an entity, or falls back to the `metadata.name` if it doesn't exist. I will use that pipe in these notifications once this is merged.